src/toil/batchSystems/lsf.py

```diff
@@ -93,21 +93,22 @@ def getJobExitCode(self, lsfJobID):
                          "{}".format(job))
             process = subprocess.Popen(args, stdout=subprocess.PIPE,
                                        stderr=subprocess.STDOUT)
-            started = 0
-            for line in process.stdout:
-                if "Done successfully" in line:
+            output = process.stdout.read().decode('ascii').replace("\n                     ", "")
+            process_output = output.split('\n')
+            for line in process_output:
```

Comment:
Should we be more explicit with the statuses (`<DONE>`, `<PEND>`, `<EXIT>`)? What happens in the current code if someone submits a job with name `DONE` to queue `PEND` and names their job `EXIT` while it is still running? An edge case, I know, but is there anything wrong with putting a check for `Status <DONE>` instead?

Reply:
I am going to submit these to our normal queue as well and get the status updates for all cases. I think we could make them a bit more specific. I like looking for `"Status <DONE>"` and I have tested it with success. I am adding `"Status "` to the beginning of the other searches as well as it appears in all of them.